### PR TITLE
docs: copy edits get started docs

### PR DESCRIPTION
**Pull request by chri2547**
This PR provides a copy edit of the Grafana Faro Web SDK get started docs.

### CODE REVIEW

File: docs/sources/tutorials/quick-start-browser.md [hunk 1]
```diff
@@ -259,7 +256,7 @@ agent.api.initOTEL(trace, context);
 ## Use
```
Comment (chri2547) [inline docs/sources/tutorials/quick-start-browser.md:256]: @domasx2 , We need to add some introductory content here to explain the context of what's to follow. I'm not sure what 'use' means.

Comment (domasx2) [reply docs/sources/tutorials/quick-start-browser.md]: added some text, wdyt?

**Review by domasx2 — APPROVED** [review]
lgtm, thanks Chris.

File: docs/sources/tutorials/quick-start-browser.md [hunk 2]
```diff
@@ -256,10 +253,12 @@ registerInstrumentations({
 agent.api.initOTEL(trace, context);
 ```
 
-## Use
+## Usage examples
+
+Below are some examples of how to use the SDK to manually push data, set user & session.
```
Comment (chri2547) [inline docs/sources/tutorials/quick-start-browser.md:258]:
```suggestion
The following examples show how to use the SDK to push data manually and set users and sessions.
```

File: docs/sources/tutorials/quick-start-browser.md [hunk 3]
```diff
@@ -119,7 +118,7 @@ const agent = initializeGrafanaAgent({
 
 ### Advanced
 
-You can also explicitly specify and customize transports and instrumentations.
+The following example shows how you can specify and customize transports and instrumentations.
```
Comment (chri2547) [inline docs/sources/tutorials/quick-start-browser.md:121]:
```suggestion
The following example shows you how to specify and customize transports and instrumentations.
```

File: docs/sources/tutorials/quick-start-browser.md [hunk 4]
```diff
@@ -70,42 +69,42 @@ integrations:
...
+   yarn add @grafana/faro-tracing-web
+   ```
 
-## 2. Initialize Grafana Javascript Agent
+## Initialize Grafana Faro Web SDK
 
-Grafana Javascript Agent has to be initialized when your web application starts.
-Several initialization snippet examples are included below.
-Choose one, add it to your application code, and customize it as required.
+Grafana Faro Web SDK must initialized when your web application starts.
```
Comment (chri2547) [inline docs/sources/tutorials/quick-start-browser.md:96]:
```suggestion
Grafana Faro Web SDK must be initialized when your web application starts.
```

File: docs/sources/tutorials/quick-start-browser.md [hunk 5]
```diff
@@ -70,42 +69,42 @@ integrations:
...
 
-#yarn
-yarn add @grafana/agent-web
-```
+   #yarn
+   yarn add @grafana/faro-web-sdk
+   ```
 
-If you want to enable [Open Telemetry](https://opentelemetry.io/docs/instrumentation/js/) based tracing, also add `@grafana/agent-tracing-web`.
+1. To enable [Open Telemetry](https://opentelemetry.io/docs/instrumentation/js/)-based tracing, run one of the following commands.
```
Comment (chri2547) [inline docs/sources/tutorials/quick-start-browser.md:84]:
```suggestion
1. To enable [Open Telemetry](https://opentelemetry.io/docs/instrumentation/js/) based tracing, run one of the following commands.
```

File: docs/sources/tutorials/quick-start-browser.md [hunk 6]
```diff
@@ -1,16 +1,15 @@
...
 ## Before you begin
 
-- Set up a Grafana Agent instance. For more information , refer to [Grafana Agent set up documentation](https://grafana.com/docs/agent/latest/set-up/).
-- Configure your instance with `app-agent-receiver` integration to expose a http collection
-  endpoint and run with the `integrations-next` flag enabled.
+- Set up a Grafana Agent instance. For more information, refer to [Set up Grafana Agent](https://grafana.com/docs/agent/latest/set-up/).
+- Configure your instance with `app-agent-receiver` integration. The integration exposes an http collection endpoint and runs with the `integrations-next` flag enabled.
 
-The following is an example of a basic Grafana Agent configuration that exposes a collector endpoint
+The following example shows a basic Grafana Agent configuration that exposes a collector endpoint
```
Comment (chri2547) [inline docs/sources/tutorials/quick-start-browser.md:10]:
```suggestion
The following example shows a basic Faro Web SDK configuration that exposes a collector endpoint
```

**Pull request merged**